Scripts/DCS-BIOS/lib/modules/aircraft_modules/MiG-29A.lua

```diff
@@ -192,6 +192,10 @@ MiG_29A:defineToggleSwitch("OXYGEN_CABIN_EMERGENCY_DECOMPRESSION_SWITCH", device
 -- Left wall
 
 -- Chut controls
```

Comment:
could you update this to "Chute" while you're here?
```suggestion
-- Chute controls
```